test/fixtures/full_config.yml

```diff
@@ -110,7 +110,7 @@ Bundler/GemComment:
   OnlyFor: []
 Bundler/GemFilename:
   Description: Enforces the filename for managing gems.
-  Enabled: false
```

Comment:
Presumably this rule would be disabled in projects using multiple Gemfiles for testing purposes?